functions/that-api-webhooks/index.js

```diff
@@ -1,15 +1,28 @@
+require('dotenv').config();
 const Sentry = require('@sentry/node');
-const dotenv = require('dotenv');
+const debug = require('debug');
+
+const dlog = debug('that:api:functions:hooks');
 
-dotenv.config();
 Sentry.init({ dsn: process.env.SENTRY_NODE_DSN });
 
 const express = require('express');
 const { auth } = require('./mw/auth');
 const docusignComplete = require('./hooks/receive/docusignComplete');
+const auth0signup = require('./hooks/receive/auth0SignUp');
 
 const app = express();
-app.use(auth);
-app.use('/docusignComplete', docusignComplete);
 
-exports.hooks = app;
+// eslint-disable-next-line no-unused-vars
+function failure(err, req, res, next) {
+  dlog('middleware catcall error %O', err);
+  Sentry.captureException(err);
+  res.set('Content-type', 'application/json').status(500).json(err.message);
```

Comment:
make it valid json.. and I belive you can just say res.status().json({}) and drop the set.

Reply:
corrected. Verified content-type is set.